syntax/scss.vim

```diff
@@ -46,6 +46,7 @@ syn cluster scssSelectors contains=@comment,cssSelectorOp,cssTagName,cssPseudoCl
 
 syn match scssProperty "\([[:alnum:]-]\)\+\s*\(:\)\@=" contained contains=css.*Prop,cssVendor containedin=cssMediaBlock nextgroup=scssAttribute,scssAttributeWithNestedDefinition
 syn match scssAttribute ":[^;]*\ze\(;\|}\)" contained contains=css.*Attr,cssValue.*,cssColor,cssFunction,cssString.*,cssURL,scssFunction,scssInterpolation,scssVariable
+syn match scssSemicolon ";$" containedin=scssDefinition
```

Comment:
I think the <code>$</code> is too restrictive because you can have, for example, one-liners where the semicolon is not at the EOL:
```scss
@mixin highlighted-background { background-color: #fc0; }
```
```suggestion
syn match scssSemicolon ";" containedin=scssDefinition
```